backend/core/database.py

```diff
@@ -898,6 +898,7 @@ def _handle_bgp_withdrawals(self):
                      'FROM hijacks LEFT JOIN bgp_updates ON (hijacks.key = ANY(bgp_updates.hijack_key)) '
                      'WHERE bgp_updates.prefix = %s '
                      'AND bgp_updates.type = \'A\' '
+                     'AND bgp_updates.timestamp >= NOW() - INTERVAL \'1 WEEK\' '
```

Comment:
FYI: I added this also here since we are not interested in very old BGP updates (time scoping)